eslint/babel-eslint-tests/test/integration/eslint/eslint.js

```diff
@@ -90,9 +13,9 @@ describe("verify", () => {
     );
   });
 
-  xit("Readable error messages (issue #3)", () => {
+  it("Readable error messages (issue #3)", () => {
     verifyAndAssertMessages("{ , res }", {}, [
-      "1:3 Parsing error: Unexpected token",
+      /1:2 Parsing error:.*Unexpected token \(1:2\)/,
```

Comment:
This error reports an absolute file path, which is why I'm assuming it was disabled. With `RegExp` support we can now enable this!